test/conformance/conformance_test.go

```diff
@@ -24,8 +24,7 @@ func TestGatewayAPIConformance(t *testing.T) {
 	opts := conformance.DefaultOptions(t)
 	opts.SkipTests = []string{
 		tests.GatewayStaticAddresses.ShortName,
-		tests.GatewayHTTPListenerIsolation.ShortName,          // https://github.com/kubernetes-sigs/gateway-api/issues/3049
-		tests.HTTPRouteBackendRequestHeaderModifier.ShortName, // https://github.com/envoyproxy/gateway/issues/3338
+		tests.GatewayHTTPListenerIsolation.ShortName, // https://github.com/kubernetes-sigs/gateway-api/issues/3049
```

Comment:
not tied to this PR, but can we fix the link, its https://github.com/envoyproxy/gateway/issues/3352